src/gui/components/workbench/workbench.styles.js

```diff
@@ -0,0 +1,29 @@
+import { css } from 'glamor'
+
+const layout = css({
+  height: '100vh',
+  display: 'grid',
+  gridTemplateColumns: 'auto 300px',
```

Comment:
you think that the sidebar width should be relative to the font size ? it seems strange, no ?